cf/src/main/scala/com/timeout/scalacloudformation/package.scala

```diff
@@ -0,0 +1,54 @@
+package com.timeout
+
+import java.time.Duration
+
+import io.circe.Json
+import enum.Enum
+
+package object scalacloudformation {
```

Comment:
We could put all that in a `ResourceAttributes` object, to map with the doc's terminology